controller_manager/test/test_load_controller.cpp

```diff
@@ -183,192 +213,314 @@ TEST_F(TestLoadController, configure_controller)
+      EMPTY_STR_VEC, "Switch with no controllers specified"),
+    std::make_tuple(
+      controller_interface::return_type::OK, STRICT, EMPTY_STR_VEC, EMPTY_STR_VEC,
+      "Switch with no controllers specified"),
+    std::make_tuple(
+      controller_interface::return_type::OK, BEST_EFFORT, EMPTY_STR_VEC,
+      EMPTY_STR_VEC, "Switch with no controllers specified"),
+    // combination of empty and non-existent controller
+    std::make_tuple(
+      controller_interface::return_type::OK, UNSPECIFIED, NONEXISTENT_CONTROLLER,
+      EMPTY_STR_VEC, "Switch with nonexistent controller specified"),
+    std::make_tuple(
+      controller_interface::return_type::ERROR, STRICT, NONEXISTENT_CONTROLLER,
+      EMPTY_STR_VEC, "Switch with nonexistent start controller specified"),
+    std::make_tuple(
+      controller_interface::return_type::OK, BEST_EFFORT, NONEXISTENT_CONTROLLER,
+      EMPTY_STR_VEC, "Switch with nonexistent start controller specified"),
+    std::make_tuple(
+      controller_interface::return_type::OK, UNSPECIFIED, EMPTY_STR_VEC,
+      NONEXISTENT_CONTROLLER, "Switch with nonexistent stop controller specified"),
+    std::make_tuple(
+      controller_interface::return_type::ERROR, STRICT, EMPTY_STR_VEC,
+      NONEXISTENT_CONTROLLER, "Switch with nonexistent stop controller specified"),
+    std::make_tuple(
+      controller_interface::return_type::OK, BEST_EFFORT, EMPTY_STR_VEC,
+      NONEXISTENT_CONTROLLER, "Switch with nonexistent stop controller specified"),
+    std::make_tuple(
+      controller_interface::return_type::OK, UNSPECIFIED, NONEXISTENT_CONTROLLER,
+      NONEXISTENT_CONTROLLER, "Switch with nonexistent start and stop controllers specified"),
+    std::make_tuple(
+      controller_interface::return_type::ERROR, STRICT, NONEXISTENT_CONTROLLER,
+      NONEXISTENT_CONTROLLER, "Switch with nonexistent start and stop controllers specified"),
+    std::make_tuple(
+      controller_interface::return_type::OK, BEST_EFFORT, NONEXISTENT_CONTROLLER,
+      NONEXISTENT_CONTROLLER, "Switch with nonexistent start and stop controllers specified"),
+    // valid controller used
+    std::make_tuple(
+      controller_interface::return_type::ERROR, STRICT, NONEXISTENT_CONTROLLER,
+      VALID_CONTROLLER, "Switch with valid stopped controller specified"),
+    std::make_tuple(
+      controller_interface::return_type::OK, BEST_EFFORT, NONEXISTENT_CONTROLLER,
+      VALID_CONTROLLER, "Switch with valid stopped controller specified"),
+    std::make_tuple(
+      controller_interface::return_type::ERROR, STRICT, VALID_PLUS_NONEXISTENT_CONTROLLERS,
+      EMPTY_STR_VEC, "Switch with valid and nonexistent controller specified"),
+    std::make_tuple(
+      controller_interface::return_type::ERROR, STRICT, VALID_CONTROLLER,
+      NONEXISTENT_CONTROLLER, "Switch with  valid and nonexistent controller specified")
+));
+#endif
+
+TEST_F(TestLoadController, starting_and_stopping_a_controller)
```

Comment:
Isn't this test done already above? I think it was "test_configure_inactive_controller".